players/Peice.java

```diff
@@ -0,0 +1,25 @@
+package players;
+
+public class Peice {
+	private int pos;
+
+    public Peice(){
+        pos = 0;
+    }
+
+    public void setPos(int newpos){
+        if(newpos >= 100){
```

Comment:
This logic should not be part of piece

Reply:
again this lopgic should be part of playing in board?

Reply:
Yes. I'll tell you the reason. Let's say you want to extend the game to 200 instead of 100 then you'll have to change at multiple places.